RPS.html

```diff
@@ -1,158 +1,96 @@
-
-  const resultDiv = document.querySelector('.result');
-  const p1 = document.createElement('p');
-  const p2 = document.createElement('p');
-  const p3 = document.createElement('p');
-
-  p1.textContent = playRound(playerSelection, computerSelection);
-  p2.textContent = "User selects: " + playerSelection;
-  p3.textContent = "Comp selects: " + computerSelection;
-
-  resultDiv.appendChild(p1);
-  resultDiv.appendChild(p2);
-  resultDiv.appendChild(p3);
-}
-
-function paperGame() {
-  let playerSelection = "Paper";
-  const computerSelection = computerPlay();
-  
-  const resultDiv = document.querySelector('.result');
-  const p1 = document.createElement('p');
-  const p2 = document.createElement('p');
-  const p3 = document.createElement('p');
-
-  p1.textContent = playRound(playerSelection, computerSelection);
-  p2.textContent = "User selects: " + playerSelection;
-  p3.textContent = "Comp selects: " + computerSelection;
-
-  resultDiv.appendChild(p1);
-  resultDiv.appendChild(p2);
-  resultDiv.appendChild(p3);
-}
-
-function scissorsGame() {
-  let playerSelection = "Scissors";
-  const computerSelection = computerPlay();
-  
-  const resultDiv = document.querySelector('.result');
-  const p1 = document.createElement('p');
-  const p2 = document.createElement('p');
-  const p3 = document.createElement('p');
-
-  p1.textContent = playRound(playerSelection, computerSelection);
-  p2.textContent = "User selects: " + playerSelection;
-  p3.textContent = "Comp selects: " + computerSelection;
-
-  resultDiv.appendChild(p1);
-  resultDiv.appendChild(p2);
-  resultDiv.appendChild(p3);
-}
-
-const rockBtn = document.querySelector('#rock');
-const paperBtn = document.querySelector('#paper');
-const scissorsBtn = document.querySelector('#scissors');
-
-rockBtn.addEventListener('click', rockGame);
-paperBtn.addEventListener('click', paperGame);
-scissorsBtn.addEventListener('click', scissorsGame);
-
- </script>
- 
-</body>
+      } else {
+        console.log("Something goes wrong...");
+      }
+    }
+
+    function game(playerSelection) {
+      const computerSelection = computerPlay();
+
+      const resultDiv = document.querySelector('.result');
+      const p1 = document.createElement('p');
+      const p2 = document.createElement('p');
+      const p3 = document.createElement('p');
+
+      p1.textContent = playRound(playerSelection, computerSelection);
+      p2.textContent = "User selects: " + playerSelection;
+      p3.textContent = "Comp selects: " + computerSelection;
+
+      resultDiv.appendChild(p1);
+      resultDiv.appendChild(p2);
+      resultDiv.appendChild(p3);
+    }
+
+    const rockBtn = document.querySelector('#rock');
+    const paperBtn = document.querySelector('#paper');
+    const scissorsBtn = document.querySelector('#scissors');
+
+    rockBtn.addEventListener('click', () => game(rock));
+    paperBtn.addEventListener('click', () => game(paper));
+    scissorsBtn.addEventListener('click', () => game(scissors));
```

Comment:
You can create another `forEach` here and replace all those lines. Similar to above.